src/Init/Lean/Elab/Command.lean

```diff
@@ -175,15 +175,20 @@ def mkCommandElabAttribute : IO CommandElabAttribute :=
 mkElabAttribute CommandElab `commandElab `Lean.Parser.Command `Lean.Elab.Command.CommandElab "command" builtinCommandElabTable
 @[init mkCommandElabAttribute] constant commandElabAttribute : CommandElabAttribute := arbitrary _
 
-def elabCommand (stx : Syntax) : CommandElabM Unit :=
-stx.ifNode
+partial def elabCommand : Syntax → CommandElabM Unit
+| stx => stx.ifNode
   (fun n => do
     s ← get;
     let table := (commandElabAttribute.ext.getState s.env).table;
     let k := n.getKind;
-    match table.find? k with
-    | some elab => elab n
-    | none      => throwError stx ("command '" ++ toString k ++ "' has not been implemented"))
+    -- list of commands => recurse
+    -- The parser will only ever return a single command, but syntax quotations can return multiple ones
```

Comment:
Please rename the original `elabCommand` function to `elabCommandAux`, and then define `elabCommand` as
```lean
def elabCommand (stx : Syntax) : CommandElabM Unit :=
if stx.getKind == nullKind then stx.getArgs.forM elabCommandAux else elabCommandAux stx
```